actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Introduce `ActionDispatch::ActionableExceptions`.
+
+    The `ActionDispatch::ActionableExceptions` middleware dispatches actions
+    from `ActiveSupport::ActionableError` descendants.
```

Comment:
We can probably expand on what this does though, or what the end goal is that we get out of this?

Reply:
I see we actually expand this below, probably a one line note here would be great.